anomalo/structs.go

```diff
@@ -139,6 +167,76 @@ type DeleteCheckResponse struct {
 	DeletedCount int `json:"deleted_count,omitempty"`
 }
 
+type RunChecksRequest struct {
+    TableID                  int      `json:"table_id,omitempty"`
+    IntervalID               int      `json:"interval_id,omitempty"`
+    CheckIDs                 []string `json:"check_ids,omitempty"`
+    Force                    bool     `json:"force,omitempty"`
+    ExecutionPriority        string   `json:"execution_priority,omitempty"`
+    RespectSkipError         bool     `json:"respect_skip_error,omitempty"`
```

Comment:
Should this be

```suggestion
    RespectSkipExpr         bool     `json:"respect_skip_expr,omitempty"`
```
?